src/pages/menuUpload/itemMaster/ItemMasterPage.tsx

```diff
@@ -30,28 +33,77 @@ import itemColumns from './columnsItem';
 const clearValues: IItem = { id: undefined, name: '', conditions: '' };
 
 function ItemMasterPage() {
+  const DEFAULT_OFFSET = 10;
+  const FIRST_PAGE = 1;
+  const EMPTY_SEARCHER = '';
+  const itemUrl = 'item/';
```

Comment:
Do we want to have const values inside the function or outside like `clearValues`?

Reply:
good idea. I can change this